.github/workflows/docs.yml

```diff
@@ -43,6 +43,14 @@ jobs:
         pip install --progress-bar=off -r requirements.txt
         echo '::endgroup::'
         
+        if [[ ${{ github.ref_type }} == tag || (${{ github.ref_type }} == branch && ${{ github.ref_name }} == release/*) ]]; then
+          echo '::group::Enable version wrangling'
+          # This environment variable just has to exist and must not be empty. The actual value is arbitrary.
+          # See docs/source/conf.py for details
+          export VERSION=1
```

Comment:
should we call this `SANITIZE_VERSION_IN_DOCS` or something like that?

Reply:
Happy to do so. Let me see if anything else relies on this specific name of the env var.

Reply:
I've `grep`'ed through the repository and it seems this env var is not used anywhere else. Meaning, we are good to go. I've used `TORCHVISION_SANITIZE_VERSION_STR_IN_DOCS`